### PR TITLE
Revert "Rook mon crashloop and pod disappearance workaround"

### PR DESCRIPTION
**Pull request by ebenahar**
Reverts red-hat-storage/ocs-ci#433

### CODE REVIEW

File: conf/ocsci/downstream_config.yaml [hunk 1]
```diff
@@ -2,4 +2,3 @@
 ENV_DATA:
   ceph_image: 'quay.io/rhceph-dev/rhceph-4.0-rhel-8:latest'
   rook_image: 'quay.io/rhceph-dev/rook:latest'
-  cluster_namespace: 'rook-ceph'
```
Comment (prsurve) [inline conf/ocsci/downstream_config.yaml]: we can keep rook-ceph namespace because openshift-storage namespace leads to pod disappear

Comment (ebenahar) [reply conf/ocsci/downstream_config.yaml]: done

File: ocs_ci/framework/conf/default_config.yaml [hunk 2]
```diff
@@ -41,11 +41,11 @@ REPORTING:
 # --cluster-conf file.yaml data you will pass to the pytest.
 ENV_DATA:
   cluster_name: null  # will be changed in ocscilib plugin
-  cluster_namespace: 'rook-ceph'
+  cluster_namespace: 'openshift-storage'
```
Comment (MeridianExplorer) [inline ocs_ci/framework/conf/default_config.yaml]: Any reason we'd like to revert to openshift-storage? 
I think it'll be better to keep it until we fix the pod disappearance bug

File: ocs_ci/ocs/defaults.py [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 INSTALLER_VERSION = '4.1.4'
 CLIENT_VERSION = INSTALLER_VERSION
-ROOK_CLUSTER_NAMESPACE = 'rook-ceph'
+ROOK_CLUSTER_NAMESPACE = 'openshift-storage'
```
Comment (MeridianExplorer) [inline ocs_ci/ocs/defaults.py]: Any reason we'd like to revert to openshift-storage? 
I think it'll be better to keep it until we fix the pod disappearance bug

**Comment by sidhant-agrawal** [comment]
@ebenahar Signed off check is failing.

File: ocs_ci/framework/conf/default_config.yaml (hunk 2 above)
Comment (ebenahar) [reply ocs_ci/framework/conf/default_config.yaml]: no, trying to change to rook-ceph

File: ocs_ci/ocs/defaults.py (hunk 3 above)
Comment (ebenahar) [reply ocs_ci/ocs/defaults.py]: same

File: conf/ocsci/downstream_config.yaml (hunk 1 above)
Comment (MeridianExplorer) [reply conf/ocsci/downstream_config.yaml]: Restored rook-ceph namespace

**Comment by sidhant-agrawal** [comment]
@MeridianExplorer Can you also update the issue, that all workaround changes were not reverted in this PR

**Comment by prsurve** [comment]
we have to hold on with this pr as cephfs pvc are not getting into bound state

**Comment by ebenahar** [comment]
@nehaberry @prsurve Can we proceed with this PR?

**Comment by prsurve** [comment]
@ebenahar yes we can proceed just tested with latest master pvc are getting into bound state checked with rbd and cephfs sc as soon as this pr get's merge we need to update yaml as per rook/msater in our ocs-ci
@nehaberry can you also confirm once

**Comment by nehaberry** [comment]
> @ebenahar yes we can proceed just tested with latest rook master image pvc are getting into bound state checked with rbd and cephfs sc as soon as this pr get's merge we need to update yaml as per rook/msater in our ocs-ci
> @nehaberry can you also confirm once

Hi Pratik,

I didnt get opportunity to test with latest branch today, but yeah it seems our yamls need to chnage the moment we revert the changes in this PR.

Following are the changes required:

1. common.yaml
2. cluster.yaml
3. operator-openshift-with-csi.yaml
4. all rbac files under csi/rbac/cephfs and csi/rbac/rbd ( need to add 2 new files in each)
5. rbd storageclass.yaml (csi/examples/rbd/storageclass.yaml) 
6. Change rook image in default_config.yaml to rook/ceph:master
7. Change  default_config.yam to add ceph image to match with cluster.yaml

Hope I didnt miss any file :) 

Thanks,
Neha

**Pull request merged**